### PR TITLE
docs(rxCapitalize): FRMW-167 Migrate rxCapitalize

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-167

- [x] Dev
- [x] Dev

### CODE REVIEW

**Comment by bry4n** [comment]
LGTM

File: src/rxCapitalize/rxCapitalize.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ angular.module('encore.ui.rxCapitalize', [])
 /**
  * @ngdoc filter
  * @name rxCapitalize.filter:rxCapitalize
- * @description [TBD]
+ * @description
+ * Filter for `rxCapitalize`
```
Comment (CITguy) [inline src/rxCapitalize/rxCapitalize.js]: I'd use the description above as the description for this filter.

```
@description
Capitalizes the first word in a string.
```

File: src/rxCapitalize/rxCapitalize.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * @description
  * # rxCapitalize Component
  *
- * [TBD]
+ * Capitalizes the first word in a string.
```
Comment (CITguy) [inline src/rxCapitalize/rxCapitalize.js]: I'd have a summary of what the component provides.

```
This component provides logic to capitalize the first word in a string via an Angular filter.
```

**Comment by CITguy** [comment]
LGTM

**Pull request merged**